Tests/AppRemoteConfigTests/SigningTests.swift

```diff
@@ -0,0 +1,73 @@
+@testable import AppRemoteConfig
+import Dependencies
+import Foundation
+import SodiumClientLive
+import XCTest
+
+@available(macOS 13, *) @MainActor
+final class SigningTests: XCTestCase {
+    
+    // $> care create-key-pair
+    // The public key is rK21qYyxsj8x75kCqU8k99zU4bEJdI60fMTPzsviBtE=.
+    // The secret key is 7/z5fFbEF1xPlCWJdqhfd6SV5lp/xusUsc7VpYTAX2asrbWpjLGyPzHvmQKpTyT33NThsQl0jrR8xM/Oy+IG0Q==.
+    
+    func testVerifying() async throws {
+        try withDependencies {
+            $0.sodiumClient = .liveValue
+        } operation: {
+            let publicKey = "rK21qYyxsj8x75kCqU8k99zU4bEJdI60fMTPzsviBtE="
+            let secretKey = "7/z5fFbEF1xPlCWJdqhfd6SV5lp/xusUsc7VpYTAX2asrbWpjLGyPzHvmQKpTyT33NThsQl0jrR8xM/Oy+IG0Q=="
```

Comment:
As this key is used multiple times below, might be better to have this as a constant

Reply:
ok check